### DISCUSSION TITLE
WhateverGreen.kext 1.5.2 seems some problem.

### DISCUSSION THREAD

**Discussion by 13months** [osy86/HaC-Mini#622] [Q&A] [answered]
using hackmini3.01 on bigsur11.15.2
After boot,it seem black
I found the problem with WhateverGreen.kext 1.5.2
OC:0.7.2

other kext is ok,but WEG 1.5.2 is not ok.

**Comment by goodbest** [comment, marked as answer]
Solution here
https://github.com/osy/HaC-Mini/issues/614#issuecomment-899037429

**Comment by 13months** [comment]
Thanks，it is OK now.艺木森你望月亮一分钟，月亮望你整整一生。邮箱：370903253------------------ Original ------------------From: goodbest  ***@***.***>Date: Tue,Aug 17,2021 9:53 PMTo: osy/HaC-Mini ***@***.***>Cc: Mu Sing ***@***.***>, Author ***@***.***>Subject: Re: [osy/HaC-Mini] WhateverGreen.kext 1.5.2 seems some problem.(#622)
Solution here
#614 (comment)

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/osy/HaC-Mini/discussions/622#discussioncomment-1195046",
"url": "https://github.com/osy/HaC-Mini/discussions/622#discussioncomment-1195046",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
